core/src/main/java/io/lionweb/lioncore/java/serialization/data/SerializedClassifierInstance.java

```diff
@@ -160,4 +164,12 @@ public String toString() {
     public void setAnnotations(List<String> annotationIDs) {
       this.annotations = annotationIDs;
     }
+
+    public String getAnnotated() {
```

Comment:
Why do we have this method? This _must_ be the same as `getParent()`, according to https://lionweb-org.github.io/organization/serialization/serialization.html#parent

Reply:
This came as a surprise to me. When I think about the model I imagine a tree (or a forest) which is composed by nodes, each one with their own position within the tree. Each one (besides roots) having a parent. Annotation instances are instances something else: they are not node, they are not part of the tree. They are attached to nodes. Some consumers are interested in the tree (so just the nodes), some others are interested in the tree and the attached annotation instances.

So it seems strange to me that an annotation instance has a parent, as it is not part of the tree. From may point of view it has a pointer to the node that it is attached to, which it is not its parent, as the annotation instance is not a children of the node (it is not part of any containment).

Reply:
The rationale was: Every node _must_ be contained in a `Partition`. We can always find the partition by moving up to the root of the containment hierarchy.
If not the annotated node, who else is the parent of an annotation instance? Which partition does it belong to?

If we wanted _completely independent_ annotations (i.e. stored somewhere outside the annotated node), they would be provided by a processor as a derived model. This processor might have its own internal storage, or ask the repository for a "special" partition that contains all the annotations.

Reply:
I think that the partition containing the annotation is the same partition which is containing the annotated node, on that we agree. My problem is with the term "parent" to indicate what it seems to me are two different relationship.

I think that the children of a node do not contain the annotations attached to that node. Is that right?

If it is right, then it is surprising that the annotation can have as a parent the annotated node. It means that parent is not the inverse relationship of children.

Reply:
That's what we decided in https://github.com/LIonWeb-org/organization/issues/150

Reply:
I hope this was _our_ understanding, not only _my_ understanding -- shall we raise the issue again in the next meeting?